docs/index.html

```diff
@@ -151,6 +156,10 @@
         generateMultipleLanguagesNavbarPluginByListItemName("Translations"),
       ],
     };
+    QABot.initialize({
+      "serverUrl": "http://81.68.181.139/ai/qa",
```

Comment:
For simulating the ai backend shutdown, 

I change 

`http://81.68.181.139` to `http://81.68.181.100`

The document can be open, but would delay 10s . 

Is there any way to setup the timeout from 10s to 5s, 3s or 2s?

Reply:
This is a good point! I changed the logic as follows and it should work well when 81.68.181.139 is down.

```javascript
<!-- add qa bot -->
<script>
  (function() {
    var link = document.createElement('link');
    link.rel = 'stylesheet';
    link.href = 'http://81.68.181.139/ai/qa-bot.css';
    document.head.appendChild(link);

    var script = document.createElement('script');
    script.src = 'http://81.68.181.139/ai/qa-bot.js';
    script.onload = function () {
      QABot.initialize({
        "serverUrl": "http://81.68.181.139/ai/qa",
        "documentSiteUrlPrefix": "https://www.apolloconfig.com/#"
      });
    };
    document.body.appendChild(script);
  })();
</script>
```

Reply:
Good Idea.

But it is still block on my machine...

Found that
```
  <link rel="stylesheet" href="http://81.68.181.139/ai/qa-bot.css">
  <script src="http://81.68.181.139/ai/qa-bot.js"></script>
```
i.e those 2 lines cause.

When change them to

```
  <link rel="stylesheet" href="http://81.68.181.100/ai/qa-bot.css">
  <script src="http://81.68.181.100/ai/qa-bot.js"></script>
```

We can reproduce it.

----

After query how to async load css and javascript through ChatGPT, There is a solution

use 
```
  <link rel="preload" href="http://81.68.181.139/ai/qa-bot.css" as="style">
  <noscript><link rel="stylesheet" href="http://81.68.181.139/ai/qa-bot.css"></noscript>
```
to async load css, and use

```
  <script src="http://81.68.181.139/ai/qa-bot.js" async></script>
```
to async load javascript.

Reply:
I'm sorry, looks like the last push failed, these 2 lines should be removed.

<link rel="stylesheet" href="http://81.68.181.139/ai/qa-bot.css">
<script src="http://81.68.181.139/ai/qa-bot.js"></script>